dev_guide/service_accounts.adoc

```diff
@@ -213,6 +213,31 @@ Tokens:            	robot-token-8bhpp
 ----
 ====
 
+== Checking if Pod Creation is Possible
+
+Using the `scc-review` and `scc-subject-review` options, you can see if an individual user, or a user under a specific service account, can create a pod.
+
+Using the `scc-review`, you can check if a service account can create a pod. The command outputs the security context constraints that admit the resource.
+
+To check if a user with the system:serviceaccount:projectname:default service account can a create a pod:
+
+----
+$  oc policy scc-review -z system:serviceaccount:projectname:default -f my_resource.yaml
+----
+
+You can also use the `scc-subject-review` option to check whether a specific user can create a pod:
+
+----
+$ oc policy scc-subject-review -u <username> -f myfile.yaml
+----
+
+To check if a user belonging to a specific group can create a pod in a specific file:
+
+----
+$ oc policy scc-subject-review -u <username> -g <groupname> -f myfile.yaml
```

Comment:
I'd suggest to use consistent naming for a files in the examples above (`my_resource.yaml` vs `myfile.yaml`).